DeferredDocumentQuery/src/main/resources/wsdl/DocumentConsumer.wsdl

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- 
+    Document Consumer Results WSDL
+  -->
+<definitions xmlns:soap="http://schemas.xmlsoap.org/wsdl/soap12/" 
+        xmlns="http://schemas.xmlsoap.org/wsdl/" 
+        xmlns:tns="urn:gov:hhs:fha:nhinc:adapterdocquerydeferredrequest"
+        xmlns:xsd="http://www.w3.org/2001/XMLSchema" 
+        xmlns:query="urn:oasis:names:tc:ebxml-regrep:xsd:query:3.0"
+        xmlns:wsu="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-utility-1.0.xsd"
+        xmlns:wsaw="http://www.w3.org/2006/05/addressing/wsdl"
+        xmlns:rs="urn:oasis:names:tc:ebxml-regrep:xsd:rs:3.0"
+        name="DocumentConsumer" 
+        targetNamespace="urn:gov:hhs:fha:nhinc:adapterdocquerydeferredrequest">
+    <documentation>Document Consumer Results</documentation>
+    <types>
+        <xsd:schema>
+            <xsd:import namespace="urn:oasis:names:tc:ebxml-regrep:xsd:query:3.0" schemaLocation="classpath:schemas/ebRS/query.xsd"/>
+            <xsd:import namespace="urn:oasis:names:tc:ebxml-regrep:xsd:rs:3.0" schemaLocation="classpath:schemas/ebRS/rs.xsd"/>
+        </xsd:schema>
+    </types>
+    <message name="RegistryStoredQueryDeferredResults_Message">
+        <part name="RegistryStoredQueryDeferredResults"
+                element="query:AdhocQueryResponse"/>
+    </message>
+    <message name="RegistryStoredQueryDeferredResultsAcknowledgement_Message">
+        <part name="RegistryStoredQueryDeferredResultsAcknowledgement"
+                element="rs:RegistryResponse"/>
+    </message>
+    <portType name="DocumentConsumer_Port_Soap12">
+        <operation name="RespondingGateway_CrossGatewayQuery">
+            <input name="RegistryStoredQueryDeferredResults_Message"
+                    message="tns:RegistryStoredQueryDeferredResults_Message"
+                    wsaw:Action="tns:DocumentConsumer_ RegistryStoredQueryDeferredResults"/>
+            <output name="RegistryStoredQueryDeferredResultsAcknowledgement_Message"
+                    message="tns:RegistryStoredQueryDeferredResultsAcknowledgement_Message" 
+                    wsaw:Action="tns:RespondingGateway_CrossGatewayQueryDeferredResponse"/>
+        </operation>
+    </portType>
+    <binding name="DocumentConsumer_Binding_Soap12" type="tns:AdapterDocQueryDeferredRequestPortType">
+        <soap:binding style="document" transport="http://schemas.xmlsoap.org/soap/http"/>
+        <operation name="DocumentConsumer_RegistryStoredQueryDeferredResults">
+            <soap:operation soapAction="DocumentConsumer_ RegistryStoredQueryDeferredResults" soapActionRequired="false"/>
```

Comment:
I was not too sure about this one - might be a typo in the document, but this was a copy/paste directly from the Final Draft.